packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/common_mappers/error_mappers/dio_error_mapper.dart

```diff
@@ -3,6 +3,9 @@ part of 'error_mapper.dart';
 extension _DioErrorMapper on DioException {
   static const String kConnectionRefusedError = 'Connection refused';
   ErrorModel asErrorModel() {
+    {{#analytics}}
+    final _ = _mapToErrorLogDetails(); // TODO: Handle details
```

Comment:
These can be included inside the error model. I haven't added them for now, since `ErrorModel` was moved to a separate package.